### PR TITLE
Feature/pop up window

### PR DESCRIPTION
**Pull request by hansderly**
In this PR, i:
- Added popup modal
- Added layout for movie image
- Added layout for comment list
- Added for form to add comment

### CODE REVIEW

**Comment by anas599** [comment]
Hi Hans,
it seems you removed the request for a review I think,
also, it shows there's a conflict between the branches

File: src/index.html [hunk 1]
```diff
@@ -97,6 +104,64 @@
...
+          close
+        </span>
+      </div>
+      <div class="modal-head-container">
+        <div class="movie-img-infos">
+          <img class="movie-img" src="https://static.tvmaze.com/uploads/images/original_untouched/189/474715.jpg"
+            alt="movie-poster" srcset="">
+          <div class="movie-infos-container">
+            <a href="#" class="movie-name">
+              GOTHAM
```
Comment (anas599) [inline src/index.html:120]: Kindly make sure the name is consumed from the API

File: src/index.html [hunk 2]
```diff
@@ -97,6 +104,64 @@
...
+          <p>Rating: <span>9.6</span></p>
+        </div>
+        <div class="language-rating">
+          <p>Release Date: 15/15/2015</p>
+          <p>Genres: Adventure</p>
+        </div>
+      </div>
+      <div class="comments-list-add-comment">
+        <div class="comment-list">
+          <h4>Comment ()</h4>
```
Comment (anas599) [inline src/index.html:137]: Kindly remove any empty comments

File: src/index.html [hunk 3]
```diff
@@ -97,6 +104,64 @@
...
+          <ul class="list">
+            <li>
+              <span class="big">1</span>
+              <div class="img-name">
+                <span>Username</span>
+              </div>
+              <span class="">This is the comment i add for testing purpose</span>
```
Comment (anas599) [inline src/index.html:144]: Kindly make sure to remove any testing

**Review by anas599 — CHANGES_REQUESTED** [review]
Great Work overall,
Please make sure to consume the data from the API regarding any information about the show such as name, image, rating, language...etc

File: src/index.html (hunk 2 above)
Comment (hansderly) [reply src/index.html:137]: Thanks for your feedback, but, That's not an empty comment, that's where I will display how many comments that specific movie has.
You can verify that in the wireframe Microverse provides.

File: src/index.html (hunk 1 above)
Comment (hansderly) [reply src/index.html:120]: That pieces of information will get from the cards you create on the homepage to make it dynamic,
That is just a placeholder for the information I will get from your cards when a user press the comment button.

File: src/index.html [hunk 4]
```diff
@@ -97,6 +104,64 @@
...
+  <div class="modal">
+    <div class="modal-content">
+      <div class="close-icon-container">
+        <span class="material-symbols-outlined">
+          close
+        </span>
+      </div>
+      <div class="modal-head-container">
+        <div class="movie-img-infos">
+          <img class="movie-img" src="https://static.tvmaze.com/uploads/images/original_untouched/189/474715.jpg"
```
[the remark this answers is not in the document]
Comment (hansderly) [reply src/index.html:116]: Thanks for your feedback, 
But That pieces of information will get from the cards you create on the homepage to make it dynamic,
That is just a placeholder for the information I will get from your cards when a user press the comment button.

[the remark this answers is not in the document]
Comment (anas599) [reply src/index.html:116]: You can use the image from the cards that I already created from the API,
Also please let me know if you will join the zoom meeting or should we rescheduled?

File: src/index.html (hunk 3 above)
Comment (hansderly) [reply src/index.html:144]: Thanks for your feedback, but, That's not an empty comment, that's where I will display how many comments that specific movie has.
You can verify that in the wireframe Microverse provides.

**Review by anas599 — APPROVED** [review]
Good Job

**Pull request merged**